src/shorthand.ts

```diff
@@ -26,7 +27,7 @@ export function parse(shorthand: string, data?, config?) {
     encoding = parseEncoding(split.join(DELIM));
 
   let spec:Spec = {
-    marktype: marktype,
+    marktype: Marktype[marktype],
```

Comment:
This is slightly ugly because we go from string to Marktype type but I think that's okay.